ChatSX/screens/FIleUpload/index.tsx

```diff
@@ -114,6 +116,7 @@ const FileUpload = ({navigation, route}: any) => {
     uploadingFilesMessages,
     isRetry,
   }: UploadResource) => {
+    LogBox.ignoreLogs(['new NativeEventEmitter']);
```

Comment:
Why this is required?

Reply:
This is a warning which comes in the new versions of react-native as many of the libraries still haven't released a new version to handle these warnings. So this is to handle that warning. Ref: https://stackoverflow.com/questions/69538962/new-nativeeventemitter-was-called-with-a-non-null-argument-without-the-requir

Reply:
Okay